### PR TITLE
Improve Terraform Registry documentation

### PR DESCRIPTION
**Pull request by johan3141592**
Added documentation on how to configure the provider to the Terraform Registry page.

### CODE REVIEW

File: templates/index.md.tmpl [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+To configure the provider to use a local user account specify the name as the provider credentials:
+```terraform
+provider "polaris" {
+  credentials = "my-account"
+}
+```
+
+## With Service Account
+First download the service account credentials as a JSON file from the Polaris User Management UI page. Next, configure the provider to use the the downloaded credentials file in the Terraform configuration:
```
Comment (wdrbk) [inline templates/index.md.tmpl:33]: Duplicate `the` in the second sentence.

File: templates/index.md.tmpl [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+The provider supports both local user accounts and service accounts. For documentation on how to create either using Polaris see the [Rubrik Support Portal](http://support.rubrik.com).
+
+## With Local User Account
+First create a directory called `.rubrik` in your home directory. Next, create a file called `polaris-accounts.json` in that directory. This JSON file holds one or more local user accounts:
+```json
+{
+  "<my-account>": {
+    "username": "<my-username>",
+    "password": "<my-password>",
+    "url": "<my-polaris-url>",
```
Comment (wdrbk) [inline templates/index.md.tmpl:19]: Perhaps not an issue in this context but strict JSON parsers won't like this trailing coma.

File: templates/index.md.tmpl [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  }
+}
+```
+Where *my-account* is an arbitary name used to refer to the account when configuring the provider. *my-username* and *my-password* are the username and password of the local user account. *my-polaris-url* is the URL of the Polaris API. The URL follows the pattern `https://{polaris-domain}.my.rubrik.com/api`. Which is the same URL as for accessing the Polaris UI but with `/api` added to the end.
```
Comment (wdrbk) [inline templates/index.md.tmpl:23]: Typo in arbitrary.

**Review by wdrbk — APPROVED** [review]

File: templates/index.md.tmpl (hunk 1 above)
Comment (johan3141592) [reply templates/index.md.tmpl]: Fixed!

File: templates/index.md.tmpl (hunk 2 above)
Comment (johan3141592) [reply templates/index.md.tmpl]: Fixed!

File: templates/index.md.tmpl (hunk 3 above)
Comment (johan3141592) [reply templates/index.md.tmpl]: Fixed!

**Pull request merged**